pages/deployment/aws.mdx

```diff
@@ -0,0 +1,180 @@
+
+
+### Setup the storage 
+
+When creating the EC2 instance, you need to set up the storage. By default
+Memgraph stores all data to working RAM, but for the [persistency between
+restarts](../fundamentals/data-durability) Memgraph uses the disk storage to
+store snapshots, configurations, etc.
+
+The sizing of the storage depends on the amount of data you are going to store
+in your Memgraph instance, and the quantity of the snapshots you want to keep
+alive. 
+
+The general rule of thumb is to double the storage and then the amount of data
+you will store. If you are going to store 100GB of data, you should have at
+least 200GB of disk storage. 
+
+
+## Installing Memgraph 
+
+Depending on the way you want to deploy Memgraph, native or via Docker, you need
+to follow the steps below: 
+
+<Tabs items={["Docker" , "Linux"]}>
+
+ <Tab> 
+    For example, if you are using Amazon Linux 2, you can follow the steps
+    below to install Docker:
+
+        ```bash
+        sudo yum update -y
+        sudo yum install docker -y
+        sudo systemctl start docker
+        # Add the user to the docker group
+        sudo usermod -a -G docker $(whoami)
+        newgrp docker
+        ```
+    Note that the steps above may vary depending on your Linux distribution.
+
+    After Docker is installed, you can pull the Memgraph image and run it:
+
+        ```bash
+        docker run -p 7687:7687 memgraph/memgraph:latest
+        ```
+
+    This will run Memgraph on the default port 7687. You should be able to connect
+    to it via [Memgraph Lab](../data-visualization) or via [client
+    libraries](../client-libraries) using the Bolt protocol. If you are experiencing issues while
+    connecting to Memgraph remotely, make sure that the port 7687 is open in [the
+    security group of your EC2 instance](#setup-the-network).
+  </Tab>
+
+    <Tab>
+
+    Depending on the Linux distribution you are using, you can follow the guide for
+    installing Memgraph on each distribution: 
+
+        - [Ubuntu](../getting-started/install-memgraph/ubuntu.mdx)
+        - [Fedora](../getting-started/install-memgraph/fedora.mdx)
+        - [Debian](../getting-started/install-memgraph/debian.mdx)
+        - [Centos](../getting-started/install-memgraph/centos.mdx)
+        - [Amazon Linux](../getting-started/install-memgraph/amazon-linux.mdx)
+        - [RedHat](../getting-started/install-memgraph/redhat.mdx)
+        - [Rocky](../getting-started/install-memgraph/rocky.mdx)
+
+    </Tab> 
+ </Tabs>
+
+## Manage Memgraph deployment
+
+After Memgraph is installed and running on your EC2 instance, Memgraph
+management in AWS is identical to the general guidelines that are described in
+the form of a native Linux Memgraph or Docker container Memgraph. 
+
+Depending on what you are using, you can follow the [Linux](./linux.mdx) or
+[Docker](./docker.mdx) deployment guide for more information on how to manage
+the Memgraph deployment.
+
+
+## Where to next? 
```

Comment:
This should be more formal

Reply:
Can you provide a bit of color on this? Except for the Kubernetes paragraph at the top, it is mostly copy-passed from other places on docs, so maybe we should fix it in other places?

Reply:
Oh. 

Yes, we should fix it in other places too, but that's a separate PR. Let's keep it consistent. I withdraw this comment at this time.